src/Interfaces/Accessibility/AccessibilityInterface.php

```diff
@@ -0,0 +1,40 @@
+<?php
+declare(strict_types=1);
+
+namespace CodeFoundation\FlowConfig\Interfaces\Accessibility;
+
+interface AccessibilityInterface
+{
+    /**
+     * They key is hidden from bulk gets by default, but can be specifically requested.
+     */
+    public const ACCESSIBILITY_HIDDEN = 0;
```

Comment:
These constants are included to provide usable keys in implementations. Should these be included or left out for the implementer to decide?